lib/active_merchant/billing/gateways/safe_charge.rb

```diff
@@ -147,6 +147,7 @@ def add_transaction_data(trans_type, post, money, options)
         post[:sg_MerchantPhoneNumber] = options[:merchant_phone_number] if options[:merchant_phone_number]
         post[:sg_MerchantName] = options[:merchant_name] if options[:merchant_name]
         post[:sg_ProductID] = options[:product_id] if options[:product_id]
+        post[:sg_NotUseCVV] = options[:not_use_cvv] ? 1 : 0 if options[:not_use_cvv]
```

Comment:
Second if makes sure that the `sg_NotUseCVV` is only sent in the request if we get in the options. Other ternary assigns value based on the boolean we get in the options. Ignoring the second if will result in passing `sg_NotUseCVV` in every request, for nil it will send as 0. do you recommend to remove the second if?

Reply:
This makes sense, I think it's fine to leave in.